xapian-applications/omega/templates/query

```diff
@@ -1,4 +1,5 @@
-$httpheader{Content-Type,text/html; charset=utf-8}<html>
+$httpheader{Content-Type,text/html; charset=utf-8}
+<html>
```

Comment:
That newline was not intentional. I just removed it now.
And I'm sorry about the commit messages that I made. I need to be more specific and careful from the next time. I will take care of the points you made. Thanks.